crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -69,6 +72,44 @@ pub async fn declare_contract(account: &str, path: &str, shortname: &str) -> Fie
     class_hash
 }
 
+pub async fn deploy_keystore_account() {
+    let keystore_path = "tests/data/keystore/deployed_key.json";
+    let account_path = "tests/data/keystore/deployed_account.json";
```

Comment:
nit: For consistency with other file names in the `keystore` directory, how about renaming:
- `deployed_key.json` -> `my_key_deployed.json`
- `deployed_account` -> `my_account_deployed.json`?

Reply:
Actually, these are two different accounts, so perhaps we should change the naming altogether? `(deployed | undeployed)_(account | key)` should be descriptive enough. wdyt?